lib/awful/menu.lua

```diff
@@ -465,14 +466,15 @@ end
 --------------------------------------------------------------------------------
 
 --- Build a popup menu with running clients and shows it.
--- @param args Menu table, see new() function for more informations
+-- @param args Menu table, see new() function for more informations.
 -- @param item_args Table that will be merged into each item, see new() for more
 --   informations.
+-- @param filter A function taking a client as an argument and returning true or
```

Comment:
Should be `@tparam func filter[opt] A function ...`.
This can be fixed when amending/merging.